app/src/main/java/org/ethereum/dappstore/data/DataRepository.kt

```diff
@@ -69,8 +69,10 @@ object DataRepository {
 
     suspend fun fetchAppById(id: String): AppInfo? {
         return apolloClient.query(AppQuery(id)).await().data?.app?.let {
-            val img = ipfs.repo.ipfs.config.base_url + "cat?arg=" + it.logo
-            val apk = ipfs.repo.ipfs.config.base_url + "get?arg=" + it.appIPFSHash
+            //val img = ipfs.repo.ipfs.config.base_url + "cat?arg=" + it.logo
+            //val apk = ipfs.repo.ipfs.config.base_url + "get?arg=" + it.appIPFSHash
+            val img = "https://gateway.pinata.cloud/ipfs/" + it.logo
```

Comment:
can you change it above as well? and make the pinata URL a constant 🙏

I think you can get rid of the `ipfs` val altogether, not using it for anything